CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ set(ENABLE_TESTS On CACHE BOOL "")
 set(ENABLE_BENCHMARKS On CACHE BOOL "")
 set(ENABLE_GMOCK On CACHE BOOL "")
 set(ENABLE_DOCS Off CACHE BOOL "")
+set(ENABLE_PEDANTIC_WARNINGS On CACHE BOOL "")
```

Comment:
Since this is our own option, it can be an `option`, rather than a `set`.

Reply:
Done.